src/components/TimeCorner/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { inject, observer } from 'mobx-react';
+import { withStyles } from '@material-ui/core';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import styles from './styles';
+import { getEndTimeCornerString } from '../../helpers';
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@inject('store')
+@observer
+export default class TimeCorner extends Component {
```

Comment:
nit: got a better name for this? Can this just be called TimeLeft?

Reply:
its the time itself, not time left, so i think its the time at the corner, so timecorner make sense.